Security/main_widget_Security_Card.yaml

```diff
@@ -222,7 +222,7 @@ slots:
                               config:
                                 accordionList: true
                                 fetchMetadata: semantics,widgetOrder,uiSemantics
-                                filter: '((loop.windowMember.category == "Window") || (loop.windowMember.category == "window")) ? true : false'
```

Comment:
Please don‘t remove the check for both spellings. When selected in the UI it will be lowercase, but some bindings define it as camelcase.